docs/request-audit-headers.md

```diff
@@ -25,28 +25,13 @@ For synchronous (i.e. non-bulk) requests, BFD users SHALL include meaningful val
 * `BlueButton-UserId`: the unique ID in the frontend system for the user (e.g. beneficiary) that data is being requested on behalf of.
 * `BlueButton-User`: the human-readable login ID (e.g. email address) in the frontend system for the user (e.g. beneficiary) that the data is being requested on behalf of.
 * `BlueButton-BeneficiaryId`: the unique beneificary ID (e.g. `Patient.id`) from BFD for the user the beneficiary whose data is being requested.
+* `x-forwarded-for`: a standard header for identifying the originating IP address of a client connecting to a web server through an HTTP proxy or a load balancer.
 
 ## Asynchronous Requests
 
 For asynchronous (i.e. bulk) requests, BFD users SHALL include meaningful values for as many as possible of the following HTTP headers.
 
-### BCDA
-
-* `BlueButton-OriginalQueryId`: (see above; same thing)
-    * For asynchronous jobs, users SHALL use the `BlueButton-OriginalQueryId` for the HTTP request that created/submitted/whatever the job.
-* `BCDA-CMSID`: the 5 digit A1234 Identifier for ACOs
-* `BCDA-JOBID`: integer generated by DB sequence
-
-### AB2D
-
-* `BlueButton-OriginalQueryId`: (see above; same thing)
-    * For asynchronous jobs, users SHALL use the `BlueButton-OriginalQueryId` for the HTTP request that created/submitted/whatever the job.
-* `AB2D-CMSID`: the Part D contract ID
-* `AB2D-JOBID`: integer generated by DB sequence
-
-### DPC
-
 * `BlueButton-OriginalQueryId`: (see above; same thing)
     * For asynchronous jobs, users SHALL use the `BlueButton-OriginalQueryId` for the HTTP request that created/submitted/whatever the job.
-* `DPC-CMSID`: The NPI of the downstream requesting provider or organization.
-* `DPC-JOBID`: integer generated by DB sequence
+* `BULK-CLIENTID`: a unique identifier of the bulk client for whom this request is for (i.e. ACO ID, NPI, Part D Contract)
```

Comment:
What is the recommended migration path for these headers?

Should BCDA start sending `BULK-CLIENTID` and `BULK-JOBID` in addition to `BCDA-CMSID` and `BCDA-JOBID`?

And if we do not send these two headers (and continue sending `BCDA-CMSID` and `BCDA-JOBID`), will our request get rejected?

Reply:
@mtrang1263 No soft migration is necessary. We can cut over in a hard fashion. These headers are used for audit purposes only. These specific bulk headers do not impact the validity of requests to BFD.